edgelet/iotedged/src/lib.rs

```diff
@@ -2237,6 +2245,30 @@ mod tests {
         }
     }
 
+    #[test]
+    fn settings_without_cert_life_uses_default() {
+        let _guard = LOCK.lock().unwrap();
+
+        let settings = Settings::new(Path::new(GOOD_SETTINGS1)).unwrap();
+        // Default is 90 days so check for that in seconds
+        assert_eq!(
+            7_776_000,
```

Comment:
`DEFAULT_AUTO_GENERATED_CA_LIFETIME_DAYS * 86_400` (and then you won't need the comment).